operator/v1/types_console.go

```diff
@@ -183,6 +187,16 @@ type QuickStarts struct {
 	Disabled []string `json:"disabled,omitempty"`
 }
 
+// AddPage allows customizing actions on the Add page in developer perspective.
+type AddPage struct {
+	// disabledActions is a list of actions that are not shown to users.
+	// Each action in the list is represented by its ID.
```

Comment:
Is there any way we can list the IDs somewhere? Either here or in the docs? As an admin changing this, I wouldn't know what to set.

@christianvogt Thoughts?

Reply:
@spadgett We provide a snippet for this in already merged PR https://github.com/openshift/console/pull/8717

Recording from the other PR:

![Peek 2021-04-21 00-21](https://user-images.githubusercontent.com/139310/115470984-8567b780-a237-11eb-9972-a94245fb3eb4.gif)

(fyi: The insertYAML automatically drops the comments.)

Reply:
We don't want to put the IDs here right now because the IDs are dynamic.
The snippet that is shown above is dynamically generated.